back/node_watcher/src/tasks/createBlockNumber.ts

```diff
@@ -17,7 +17,7 @@ const l = logger('Task: BlockNumber');
  */
 const createBlockNumber: Task<NomidotBlock> = {
   name: 'createBlockNumber',
-  read: async (blockHash: Hash, api: ApiPromise): Promise<NomidotBlock> => {
+  read: async (blockHash: Hash, events: EventRecord[], api: ApiPromise): Promise<NomidotBlock> => {
```

Comment:
btw, we started with `Task` having `read` and `write`. When i have a look at the tasks, it seems that read/write could be merged together, the distinction is not always relevant

Reply:
hmm,  I think for now the separation is a helpful way of organizing the code at least. But merging might actually be necessary anyway for the task parallelization PR down the road. I'd leave this for another PR